google/cloud/bigquery/client.py

```diff
@@ -3163,6 +3163,7 @@ def query(
         project: str = None,
         retry: retries.Retry = DEFAULT_RETRY,
         timeout: float = None,
+        job_retry: retries.Retry = None,
```

Comment:
Do we want `None` to be a way to explicitly disable retries? It could be confusing if someone doesn't want jobs to retry for some reason.

Reply:
OK, I'll make None work to disable.

Reply:
Done